src/v/kafka/server/handlers/describe_acls.cc

```diff
@@ -88,7 +75,14 @@ ss::future<response_ptr> describe_acls_handler::handle(
 
     try {
         auto filter = details::to_acl_binding_filter(request.data);
-        fill_response(ctx, filter, data);
+        auto result = co_await ctx.security_frontend().describe_acls(
+          filter, default_describe_acls_timeout);
+        if (result.error != cluster::errc::success) {
+            vlog(klog.debug, "Error describing ACLs: {}", result.error);
+            data.error_code = map_topic_error_code(result.error);
```

Comment:
i'd consider making the timeout a bit longer like 5 seconds or something. and i wonder if we were to experience a timeout if we should just return what we have locally. mmm, i guess maybe not. btw, what do we return? shoudl we return not leader or something like that?